portfolio1.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+
+<html>
+    
+    <head>
+
+        <title>About Me</title>
+
+        <meta charset="utf-8"/>
+
+        <link rel="stylesheet" type="text/css" href="portfolio1.css">
+
+    </head>
+
+    <body>
+        
+        <hr>
+
+        <div class="container"></div>
+
+        <h2>About Me <a href= "portfolio2.html">Portfolio</a></h2>
+            
+        <h1>John Doe</h1>
+
+        </div>
+    
+        <hr>
+        
+        <h3>I enjoy life being a developer</h3>
+        
+        <p>I'm John Doe, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
+        
+        <img 
+        src="images/joekaws.jpg"
```

Comment:
@tk9285 your file path is broken, instead of lowercase images, your actual file is named "Images" :hammer: needs to be fixed for images to render.